public/app/plugins/datasource/dashboard/plugin.json

```diff
@@ -2,7 +2,19 @@
   "type": "datasource",
   "name": "-- Dashboard --",
   "id": "dashboard",
-
   "builtIn": true,
+
+  "info": {
```

Comment:
all the "-- XXXX --" datasources are a mess 😬 

I suppose filling this in is the easiest approach, but it should never be used or shown

Reply:
i think there's the `hideFromList` option? Maybe that's something else

either way, yeah, this change is purely in pursuit of uniformity and satisfying pluginmeta schema as it currently stands